source/javascripts/components/ProductTour/Highlighter.tsx

```diff
@@ -48,15 +48,17 @@ export const Highlighter = ({
 						<Box
 							position="absolute"
 							backgroundColor= "transparent"
-							borderRadius="1"
+							borderRadius="8"
+							border="10px solid white"
 							boxShadow="0 0 0 3px rgba(151, 71, 255, 1)"
 							zIndex="200"
 							style={{
-								left: `${rect?.x}px`,
-								top: `${rect?.y}px`,
-								width: `${rect?.width}px`,
-								height: `${rect?.height}px`
+								left: `${rect?.x - 10}px`,
```

Comment:
why do you need these offset values (`-10`) instead of passing in the desired value as it was in the previous implementation? Is it the addition of 10px border?

Reply:
Yes, exactly. It helps to add the thick white border around each highlighted item.